internal/router/router.go

```diff
@@ -22,7 +22,7 @@ import (
 	"context"
 
 	sidecar "github.com/kubeslice/router-sidecar/pkg/sidecar/sidecarpb"
-	kubeslicev1beta1 "github.com/kubeslice/worker-operator/api/v1beta1"
+	meshv1beta1 "github.com/kubeslice/worker-operator/api/v1beta1"
 	"google.golang.org/grpc"
```

Comment:
why do we rename `kubeslicev1beta1` to `meshv1beta1`?

Reply:
changed it